src/main/java/net/atos/zac/app/planitems/PlanItemsRESTService.java

```diff
@@ -174,7 +175,7 @@ public void doHumanTaskplanItem(final RESTHumanTaskData humanTaskData) {
         final Date fataleDatum;
         if (humanTaskData.taakdata.containsKey(TAAK_DATA_DOORLOPTIJD)) {
             fataleDatum = DateTimeConverterUtil.convertToDate(humanTaskData.taakdata.get(TAAK_DATA_DOORLOPTIJD));
-            if (humanTaskData.taakdata.containsKey(TAAK_DATA_OPSCHORTEN) && humanTaskData.taakdata.get(TAAK_DATA_OPSCHORTEN).equals("true")) {
+            if (humanTaskData.taakdata.containsKey(TAAK_DATA_OPSCHORTEN) && StringUtils.equals(humanTaskData.taakdata.get(TAAK_DATA_OPSCHORTEN), "true")) {
```

Comment:
Als een null pointer exception het probleem was, waarom je dit doet. Dan is het eenvoudiger om gewoon de equals om te draaien, de string "true" kan immers nooit null zijn. En het contract van equals ken je (uit Effective Java)... toch?

Reply:
Klopt, maar op andere plekken gebruiken we voor dergelijke null-safe vergelijkingen ook StringUtils. De meeste van die wijzigingen zijn door Andy geintroduceerd, er was wel een goede reden technische reden voor.

Reply:
Nou achterhaal die reden dan maar. Je bent hier om te leren. Ik denk dat de reden was "dat werkt altijd, zelfs als beide kanten null kunnen zijn" of "ik denk er liever niet over na, maar doe gewoon altijd hetzelfde". In het geval van een string constante is het onzin.

Reply:
Opgelost (omgedraaid) in opvolgende commit